src/icons/scripts/convert.ts

```diff
@@ -42,6 +44,43 @@ const COMPONENT_PATH = path.resolve(__dirname, "..", "..", "..", "icons");
               path.basename(filename, path.extname(filename))
             );
 
+            function makeClassName() {
```

Comment:
nit: it seems like this could just be a JS object instead of a function?  also it's making a _css_ property not so much a classname so perhaps `const svgCss = "...";  const cssProp = {...};`